tools/devtool

```diff
@@ -227,10 +228,12 @@ build_bin_path() {
 cmd_fix_perms() {
     # Yes, running Docker to get elevated privileges, just to chown some files
     # is a dirty hack.
-    run_devctr \
-        --workdir "$CTR_FC_ROOT_DIR" \
-        -- \
-        chown -R "$(id -u):$(id -g)" "$CTR_FC_BUILD_DIR"
+    for dir in "$CTR_FC_BUILD_DIR" "$CTR_TEST_RESULTS_DIR"; do
+      run_devctr \
+          --workdir "$CTR_FC_ROOT_DIR" \
+          -- \
+          chown -R "$(id -u):$(id -g)" "$dir"
```

Comment:
`chown` accepts multiple arguments, so we can avoid the loop (and Docker overhead) with 

        chown -R "$(id -u):$(id -g)" "$CTR_FC_BUILD_DIR"  "$CTR_TEST_RESULTS_DIR"

Reply:
Oh, neat! I'll probably extract this commit into a separate PR tbh, I cannot for the life of it figure out the CI failures on this refactor :sob:

Reply:
https://github.com/firecracker-microvm/firecracker/pull/4800 :)